generate.py

```diff
@@ -101,32 +101,29 @@ def linkify(text):
 	return text
 
 def generate_toc(sections):
-	toc = ["<h2 id=\"toc\">Table of Contents</h2>", "<ul>"]
+	toc = ["<div id=\"toc\">"]
 	for s in sorted(sections):
 		# Add the Section Header.
-		toc.append("<li>")
-		toc.append(generate_link(split_section(s)[0] + " - " + s))
+		toc.append("<h3>"  + s + "</h3>")
```

Comment:
I think the section headers should be a link, so people can jump directly to the section.